examples/Retrieval-Models.ipynb

```diff
@@ -0,0 +1,1528 @@
+{
```

Comment:
<p>This should be updated to unique_by_tag() when  <a href="https://github.com/NVIDIA-Merlin/models/pull/267" target="_blank">PR #267</a> is merged</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>